packages/gatsby-source-nacelle/package.json

```diff
@@ -27,9 +27,14 @@
     "lodash.set": "^4.3.2"
   },
   "peerDependencies": {
-    "gatsby": ">2.0.31",
-    "gatsby-source-filesystem": "^2.6.1",
-    "graphql": "^14.7.0"
+    "gatsby": "^4.0.0",
+    "gatsby-source-filesystem": "^4.0.0",
+    "graphql": "^15.0.0"
+  },
+  "peerDependenciesMeta": {
+    "gatsby-source-filesystem": {
```

Comment:
I made this optional because we have code that is supposed to check for it, but I'm not sure if that's totally the right move or not here.